tests/tests_pytorch/helpers/datamodules.py

```diff
@@ -54,7 +54,7 @@ def test_dataloader(self):
 class SklearnDataModule(LightningDataModule):
     def __init__(self, sklearn_dataset, x_type, y_type, batch_size: int = 10):
         if not _SKLEARN_AVAILABLE:
-            pytest.skip("`sklearn` is not available.")
+            pytest.skip("`scikit-learn` is not available.")
```

Comment:
This will include the reason

```suggestion
            pytest.skip(str(_SKLEARN_AVAILABLE)):
```